schema/workflow.json

```diff
@@ -26,7 +26,8 @@
     "version": {
       "type": "string",
       "description": "Workflow version",
-      "minLength": 1
+      "minLength": 1,
+      "pattern": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
Is this the regex recommended on the semver website?

Reply:
Yes, it's the second one here: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string